docs/en/observability/logs-checklist.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[logs-checklist]]
+= Logs resource guide
+
+In this guide, you'll find resources on sending log data to {es}, configuring your logs, and analyzing your logs. 
+
+[discrete]
+[[logs-getting-started-checklist]]
+== Get started with logs
+
+If you're new to ingesting, viewing, and analyzing logs with Elastic, see <<logs-metrics-get-started, Get started with logs and metrics>> for an overview of adding integrations, installing and running an {agent}, and monitoring logs.
+
+[discrete]
+[[logs-send-data-checklist]]
+== Send data to {es}
+
+You can send data to {es} in different ways depending on your needs: 
+
+- {agent}
+- {filebeat} 
+
+When choosing between {agent} and {beats}, consider the different features and functionalities between the two options. 
+See {fleet-guide}/beats-agent-comparison.html[{agent} and {beats} capabilities] for more information which option best fits your situation.
+
+[discrete]
+[[agent-ref-guide]]
```

Comment:
I don't think we need this here. The user comes here to read get his logs files. So the user should have something like "Docker logs" and we tell them what to do in this scenario. A user here likely doesn't even know what Elastic Agent is and we expose them already to many options.

Reply:
That's the direction I want the new docs to take. With this guide, I was just hoping to help people find information that they might not know exists because things are spread out so widely in docs. As we create the new docs, I want to focus mainly on accomplishing specific logging tasks, not so broad as what the different shippers do.

Reply:
SGTM